pythonforandroid/bootstraps/qt/build/templates/AndroidManifest.tmpl.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+      package="{{ args.package }}"
```

Comment:
A recent change, require to not have `package`:
https://github.com/kivy/python-for-android/pull/2887